cbpro/authenticated_client.py

```diff
@@ -173,6 +173,45 @@ def get_account_holds(self, account_id, **kwargs):
         endpoint = '/accounts/{}/holds'.format(account_id)
         return self._send_paginated_message(endpoint, params=kwargs)
 
+    def get_profiles(self, **kwargs):
+        """List Profiles
+        
+        List all account profiles (also known as portfolios).
+        
+        Returns:
+        [
+            {
+                "id": "86602c68-306a-4500-ac73-4ce56a91d83c",
+                "user_id": "5844eceecf7e803e259d0365",
+                "name": "default",
+                "active": true,
+                "is_default": true,
+                "created_at": "2019-11-18T15:08:40.236309Z"
+            }
+        ]
+        """
+        return self._send_message('get', '/profiles')
```

Comment:
`GET /profile` has a parameter `active` so we should add some logic in case this parameter is provided (pass as keyword argument `data` to `_send_message()`